src/shared/conf-parser.c

```diff
@@ -679,6 +678,45 @@ int config_parse_many(
         return 0;
 }
 
+int config_parse_standard_file_with_dropins_full(
```

Comment:
"standard" is such a weird word for this...

Reply:
I think it's "standard" for us… I don't care for the name particularly, suggestions welcome.